src/dev-app/BUILD.bazel

```diff
@@ -158,10 +160,36 @@ dev_server(
         "@npm//:node_modules/tslib/tslib.js",
         "@npm//:node_modules/zone.js/dist/zone.js",
     ],
+)
+
+dev_server(
```

Comment:
`dev_server` should also be tagged as `manual` correct?

It might make more sense for it to be marked as such in the defintion of it at `//tools/dev-server:index.bzl`

Reply:
Yeah, good catch 👍 . I'll do that in a separate PR as we should audit other targets too and this PR is not about the `dev-server` (but the dev-app 😁 ).